src/gateways/S3Gateway.ts

```diff
@@ -49,7 +49,8 @@ export class S3Gateway {
           Conditions: [
             { bucket: this.bucketName },
             ['starts-with', '$key', `${documentId}/`],
-            { 'X-Amz-Server-Side-Encryption': 'AES256' }
+            { 'X-Amz-Server-Side-Encryption': 'AES256' },
+            ['starts-with', '$X-Amz-Meta-Description', '']
```

Comment:
Wouldn't everything start with '' ? 🤔 Is it just making sure it's not null?

Reply:
Yep, this is just making sure the value is set, even empty strings are allowed - for cases where there's no description.